src/components/Answers.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Answers = (props) => {
+  const { id, answers } = props;
+
+  return(
+    <article className="answers">
+      <input
```

Comment:
Thanks! This morning I was going to research how to make the radio buttons work properly.